src/query.go

```diff
@@ -169,7 +170,12 @@ func ParseQuery(rs query) (*query, error) {
 		s = s[i:]
 
 		q.Query = s
-		if q.tp == Q_UNKNOWN || q.tp == Q_COMMENT_WITH_COMMAND {
+		switch q.tp {
+		case Q_COMMENT_WITH_COMMAND:
+			if err := q.getQueryType(s); err != nil {
```

Comment:
shoud use `s` or `realS`?

Reply:
If I use `realS`, it will be unavailable for this functions.

Reply:
```Q_COMMENT_WITH_COMMAND``` will remove the ```--``` prefix and save command name into `s`.